### ISSUE TITLE
Emojis in aria-labels not read in some situations

### ISSUE THREAD

**Issue by bitsurfr** [FreedomScientific/VFO-standards-support#521]
### Summary

In a web browser (reproduces on Chrome, Firefox, and Edge), navigating a contenteditable using the up and down arrow keys does not read emojis in aria-labels, but using the left and right keys does.

This occurs in JAWS 2020.2006.12, but not JAWS 2019.1912.1.

1. Enter the top of a contenteditable.
2. Arrow down to have an entire row read. Observe "graphic" is announced when encountering HTML like `
<img aria-label="😍" src="url">`
3. Use the left/right arrows to navigate between items and the emoji symbol will be read. E.g. "smiling face graphic".

### Expected result

If an emoji is in an aria-label it would consistently be announced. This is used to provide an accessible backup when rendering emojis using inline images.

### Actual result

The aria-label was inconsistently announced.

### Example

https://jsbin.com/jumebofeku/edit?html,output

#### JAWS version and build number
JAWS 2020.2006.12
 
#### Operating System and version

Windows 10 Enterprise

#### Browser and version:

Chrome 90.

**Comment by JAWS-test** [comment]
The problem also occurs with JAWS 2021.

Although the line contains only 3 graphics, JAWS outputs 7 graphics (without labels, only one unknown character is recognized, which is displayed as a square in the Braille Viewer).

It should be mentioned that the graphics are located within role=textbox
